src/environments/environment.ci.ts.template

```diff
@@ -25,6 +25,8 @@ const ADMIN_API = '<%= ADMIN_API_BASE %>';
 const ECD_API = '<%= ECD_API_BASE %>';
 const biologicalScreeningDeviceAPI = `${ADMIN_API}diagnostics/biologicalScreeningDevice`;
 const sessionStorageEncKey = '<%= SESSION_STORAGE_ENC_KEY %>';
+const FHIR_API = '<%= FHIR_API %>';
```

Comment:
How have we defined FHIR api env variable in other UI files?
Is this FHIR_API or FHIR_API_BASE? This must be added to ci-prebuild.js also.